code/parse/sexp.cpp

```diff
@@ -615,6 +615,7 @@ SCP_vector<sexp_oper> Operators = {
 	{ "get-collision-group",			OP_GET_COLGROUP_ID,						1,	1,			SEXP_ACTION_OPERATOR,	},
 	{ "change-team-color",				OP_CHANGE_TEAM_COLOR,					3,	INT_MAX,	SEXP_ACTION_OPERATOR,	},	// The E
 	{ "replace-texture",				OP_REPLACE_TEXTURE,						3,  INT_MAX,	SEXP_ACTION_OPERATOR,   },  // Lafiel
+	{ "replace-texture-skybox",				OP_REPLACE_TEXTURE_SKYBOX,						2, 2,	SEXP_ACTION_OPERATOR,   },  // Lafiel
```

Comment:
My preference is `replace-skybox-texture` but I'll defer to whatever you and wookieejedi like.

Reply:
Ah yeah, following the format of the other skybox sexps (``"set-skybox-orientation" `"set-skybox-alpha"`) I see how this makes sense.

Reply:
okay